### ISSUE TITLE
Portable Generator Config (Cross-Mod) specifically with Pneumaticcraft

### ISSUE THREAD

**Issue by RyeWhit** [TwistedGate/ImmersivePetroleum#27]
### Description of the issue
Adding additional fuels to the config file for Immersive Petroleum "Portable Generator" fuels and "Motorboat" fuels did nothing to the FuelRegistry.  
Specifically I was working with PneumaticCraft Gasoline attempting to provide 1 for 1 FE generation with the Portable Generator.  Noticed in current modpack dev environment when testing before server updates that the changes to the config:
[Generation]
	#List of Portable Generator fuels. Format: fluid_name, mb_used_per_tick, flux_produced_per_tick
	fuels = ["immersivepetroleum:gasoline, 5, 256", "pneumaticcraft:gasoline, 5, 256"]
Did not seem to affect in game testing.

  I further tested with only the core mods.

### Versions and Modlist
No crash log.
Mods:
  [✔️] ImmersiveEngineering-1.16.5-5.0.0-135
  [✔️] ImmersivePetroleum-1.16.5-3.3.0-4
  [✔️] pneumaticcraft-repressurized-1.16.5-2.12.5-190
Forge 1.16.5-36.10

I was eventually able to change the FuelRegistry direct by studying the API and using CraftTweaker and some testing.
With CraftTweaker 1.16.5-7.1.0.304  I was able to modify your script from the github Wiki reference 1.14+ from 1.16+

import crafttweaker.api.fluid.IFluidStack;
mods.immersivepetroleum.FuelRegistry.registerGeneratorFuel(<fluid:pneumaticcraft:gasoline> * 256, 5); // 256fluxgen, Using 5mB per Tick

TLDR; This script made the valid FuelRegistry for Gasoline compatibility.  The config did not.
